### ISSUE TITLE
button created by tota11y.min.js does not have a name

### ISSUE THREAD

**Issue by Rayraegah** [Khan/tota11y#97]
**WCAG2AAA.Principle4.Guideline4_1.4_1_2.H91.Button.Name**
_First result_: This button element does not have a name available to an accessibility API. Valid names are: title attribute, element content.

**Selector**: `#tota11y-toolbar > button` 
**Context**:` <button aria-controls="tota11y-toolbar" class="tota11y-toolbar-toggle" aria-label="[tota11y] Toggle menu"><div aria-hidden="true" class="...</button>`

**Comment by jdan** [comment]
Thanks @Rayraegah,

The `aria-label` should do exactly this, right? 

![image](https://cloud.githubusercontent.com/assets/287268/17774367/4edccc58-6521-11e6-8648-8a34bd52331e.png)

Would love to hear more about your auditing tool :)
